ZWaveLib/Enums/EventParameter.cs

```diff
@@ -78,7 +78,9 @@ public enum EventParameter
         SecurityGeneratedKey,
         DoorLockStatus,
         RoutingInfo,
-        Clock,
+        Clock,
+        CentralScene,
+        CentralSceneCount,
```

Comment:
Don't you mind to rename this to CentralSceneNotification and CentralSceneSupportedReport?